src/i18n/Engine.cpp

```diff
@@ -59,16 +59,53 @@ I18n::CI18nEngine::CI18nEngine() {
 
+    // de_CH (Swiss German)
+    huEngine->registerEntry("de_CH", TXT_KEY_ANR_TITLE, "Aawändig Reagiert Ned");
+    huEngine->registerEntry("de_CH", TXT_KEY_ANR_CONTENT, "D'Aawändig <b>{title} - {class}</b> reagiert ned.\nWas wend Sie demet mache?");
+    huEngine->registerEntry("de_CH", TXT_KEY_ANR_OPTION_TERMINATE, "Beände");
+    huEngine->registerEntry("de_CH", TXT_KEY_ANR_OPTION_WAIT, "Warte");
+    huEngine->registerEntry("de_CH", TXT_KEY_ANR_PROP_UNKNOWN, "(onbekannt)");
+
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_REQUEST_UNKNOWN, "D'Aawändig <b>{app}</b> fordert en onbekannti Berächtigong aa.");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_REQUEST_SCREENCOPY, "D'Aawändig <b>{app}</b> versuecht Ehre Beldscherm uufznäh.\n\nWend Sie das erlaube?");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_REQUEST_PLUGIN, "D'Aawändig <b>{app}</b> versuecht es Plugin z'lade: <b>{plugin}</b>.\n\nWend Sie das erlaube?");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_REQUEST_KEYBOARD, "En neui Tastatur esch erkönnt worde: <b>{keyboard}</b>.\n\nWend Sie die in Betreb nä?");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_UNKNOWN_NAME, "(onbekannt)");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_TITLE, "Berächtigongsaafrog");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_PERSISTENCE_HINT, "Tip: Sie chönd permanenti Regle deför i Ehrere Hyprland-Konfigurationsdatei festlegge.");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_ALLOW, "Erlaube");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_ALLOW_AND_REMEMBER, "Erlaube ond merke");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_ALLOW_ONCE, "Einisch erlaube");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_DENY, "Verweigere");
+    huEngine->registerEntry("de_CH", TXT_KEY_PERMISSION_UNKNOWN_WAYLAND_APP, "Onbekannti Aawändig (wayland client ID {wayland_id})");
+
+    huEngine->registerEntry(
+        "de_CH", TXT_KEY_NOTIF_EXTERNAL_XDG_DESKTOP,
+        "Ehri XDG_CURRENT_DESKTOP omgäbig schiint extern gmanagt z'wärde, ond de aktuelli Wärt esch {value}.\nDas chönnt zo Problem füehre sofärn das ned absechtlech so esch.");
+    huEngine->registerEntry("de_CH", TXT_KEY_NOTIF_NO_GUIUTILS,
+                            "Ehres System hed hyprland-guiutils ned installiert. Das esch för es paar Dialogfänster. Es werd empfohle sie z'installiere.");
+    huEngine->registerEntry("de_CH", TXT_KEY_NOTIF_FAILED_ASSETS,
+                            "Hyprland hed {count} essentielli Ressource ned chönne lade, gäbed Sie im Packager vo Ehrere Distribution schold för es schlächts Package!");
+    huEngine->registerEntry("de_CH", TXT_KEY_NOTIF_INVALID_MONITOR_LAYOUT,
+                            "Ehres Beldschermlayout esch fählerhaft uufgsetzt. De Beldscherm {name} öberlappt met andere Beldscherm(e) im Layout.\nBitte lueged sie im Wiki "
```

Comment:
```suggestion
                            "Ehres Beldschermlayout esch fählerhaft uufgsetzt. De Beldscherm {name} öberlappt met andere Beldscherm(e) im Layout.\nBitte lueged Sie im Wiki "
```

you missed this one, otherwise all of @WhySoBad's corrections are here now thanks!

(you might have to do a rebase tho cause you moved the section)